src/containers/settings/EditServiceScreen.js

```diff
@@ -206,8 +206,11 @@ class EditServiceScreen extends Component {
         },
         trapLinkClicks: {
           label: intl.formatMessage(messages.trapLinkClicks),
-          value: service.trapLinkClicks,
-          // TODO: THIS ONLY WORKS AS A DEFAULT FOR THE CREATION OF A RECIPE. IT DOESN'T OVERRIDE THE CURRENT EXISTING RECIPES SETTINGS FOR NOW (IT SHOULD).
+          // THIS OVERRIDES THE CURRENT EXISTING RECIPES SETTINGS IF THE GLOBAL SETTING IS SET TO TRUE. 
+          // TODO: SOLUTION FOR WHEN GLOBAL SETTING IS TRUE AND THE USER WANTS TO SET A SPECIFIC SERVICE TO OFF.
+          value: isTrapLinkClicksDefault
+                  ? isTrapLinkClicksDefault
+                  : service.trapLinkClicks,
```

Comment:
Can this be easier written as: `value: isTrapLinkClicksDefault || service.trapLinkClicks,`?